draft-ietf-quic-v2.md

```diff
@@ -145,9 +145,9 @@ with clients. In particular, HTTP clients often use Alt-Svc {{?RFC7838}} to
 discover QUIC support. As this mechanism does not currently distinguish between
 QUIC versions, HTTP servers that support multiple versions reduce the
 probability of incompatibility and the cost associated with QUIC version
-negotiation or TCP fallback. For example, an origin advertising support for "h3" in Alt-Svc
-SHOULD support QUIC version 1 as it was the original QUIC version used by HTTP/3 and
-therefore some clients will only support that version.
+negotiation or TCP fallback. For example, an origin advertising support for "h3"
+in Alt-Svc SHOULD support QUIC version 1 as it was the original QUIC version
```

Comment:
You have two CLs in flight with this no-op change. In the future I'd suggest just fixing that directly on main to simplify reviews